src/apis/post.ts

```diff
@@ -8,7 +15,17 @@ export interface submitPostProps {
 
 async function submitPost(param: submitPostProps) {
   const response = await instance.post('/posts', JSON.stringify(param));
+  return response.status;
+}
+
+async function updatePost(id: string, param: submitPostProps) {
+  const response = await instance.put(`/posts/${id}`, JSON.stringify(param));
+  return response.status;
+}
+
+async function deletePost(id: number) {
+  const response = await instance.delete(`/posts/${id}`);
```

Comment:
delete 한 것도 response 라는 변수로 할 필요가 있을까용 ???
```suggestion
async function deletePost(id: number) {
  await instance.delete(`/posts/${id}`);
```

Reply:
응답값을 리턴하는 이유는 삭제가 성공되었을때 status 코드를 가지고 성공여부를 판별합니다. 해당 값 정상적으로 리턴이 된후 메인페이지로 리다이렉트 하도록 작업하였기 떄문입니다.